BasicSample/Assets/ARAnchor/ARAnchor.unity

```diff
@@ -395,13 +617,10 @@ TextMesh:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 664740217}
-  m_Text: 'AR Anchors and Anchor Persistence
-
-
-    Air tap anywhere to create a free-world
-    anchor.
-
-    Air tap near an anchor to toggle persistence.'
+  m_Text: "AR Anchors and Anchor Persistence\n\nAir tap anywhere to create a free-world
```

Comment:
You might find the text becomes crowed when adding more words to the instruction panel.  Might need to tweak the font size.  Once using the "auto sized" text panel, it will become easier.  Reference to PR https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/pull/76